aviary/mission/gasp_based/ode/climb_ode.py

```diff
@@ -48,10 +48,10 @@ def setup(self):
 
         if input_speed_type is SpeedType.EAS:
             speed_inputs = ["EAS"]
-            speed_outputs = ["mach", "TAS"]
+            speed_outputs = ["mach", ("TAS", Dynamic.Mission.VELOCITY)]
         elif input_speed_type is SpeedType.MACH:
             speed_inputs = ["mach"]
-            speed_outputs = ["EAS", "TAS"]
+            speed_outputs = ["EAS", ("TAS", Dynamic.Mission.VELOCITY)]
 
```

Comment:
Now that add_flight_conditions has the correct inputs/outputs, could that be used here?

Reply:
I looked at this a bit; it could but I think it's better to have it explicitly spelled out in this case because the `flight_condition_group` is dependent on analysis_scheme and having it in one place here will make it easier for people to understand:

```
            flight_condition_group = mach_balance_group

        elif analysis_scheme is AnalysisScheme.SHOOTING:
           <...>
            flight_condition_group = self
```